pkg/pillar/cmd/zedmanager/zedmanager.go

```diff
@@ -897,6 +945,14 @@ func handleZedAgentStatusImpl(ctxArg interface{}, key string,
 	statusArg interface{}) {
 	ctxPtr := ctxArg.(*zedmanagerContext)
 	status := statusArg.(types.ZedAgentStatus)
+	if status.ConfigGetStatus == types.ConfigGetSuccess {
```

Comment:
Success or Read; both means the device has a valid config.

Reply:
Ah, yep, thanks!
Fixed